int32 id
---
int32 count
float32[] x
float32[] y
int32[] priority